### ISSUE TITLE
Rename project

### ISSUE THREAD

**Issue by led02** [hermes-hmc/workflow#29]
"workflow" is not very speaking and produces clashes with other directory.

**Comment by sdruskat** [comment]
We talked about renaming it to `hermes-cli` (and actually also renaming `haggis` to `hermes_cli`).

**Comment by led02** [comment]
> We talked about renaming it to `hermes-cli` (and actually also renaming `haggis` to `hermes_cli`, `hermescli` or something similar).

I also think that a more obvious name could be helpfult for getting people started. Alternative, one could also have something like `metadata-pub` (e.g., use a "verb" in best Unix tradition...)

**Comment by poikilotherm** [comment]
I agree on the idea of subject + verb thing, too. Yet I would go with a name offering broader scope, like `software-pub` or shorter `swpub`. (I'm lazy, I like short.)

Or a fancy combo! Does `swpubcli` resonate with you?

**Comment by sdruskat** [comment]
> I agree on the idea of subject + verb thing, too. Yet I would go with a name offering broader scope, like `software-pub` or shorter `swpub`. (I'm lazy, I like short.)
> 
> Or a fancy combo! Does `swpubcli` resonate with you?

In theory I like it, in practice I think it doesn't improve the situation apart from getting rid off the humoristic level of `haggis`. I'd rather stick with something `*hermes*` to keep the cognitive links between project, repo org and tooling. My .2ct.

**Comment by sdruskat** [comment]
We decided to just call this `hermes` and be done with it.

**Comment by led02** [comment]
It remains to rename the GitHub project (maybe?) Otherwise, we should close this issue! :)

**Comment by sdruskat** [comment]
Closing as done.
